datafusion/sql/tests/sql_integration.rs

```diff
@@ -15,6 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+use core::panic;
```

Comment:
I haven't seen `panic` explicitly imported before (I think panic is in the standard prelude) 🤔

Reply:
```suggestion
```